hazelcast-enterprise/server.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+PRG="$0"
+PRGDIR=`dirname "$PRG"`
+HAZELCAST_HOME=`cd "$PRGDIR/.." >/dev/null; pwd`/
+PID_FILE=$HAZELCAST_HOME/hazelcast_instance.pid
+
+if [ "x$MIN_HEAP_SIZE" != "x" ]; then
+	JAVA_OPTS="$JAVA_OPTS -Xms${MIN_HEAP_SIZE}"
+fi
+
+if [ "x$MAX_HEAP_SIZE" != "x" ]; then
+	JAVA_OPTS="$JAVA_OPTS -Xmx${MAX_HEAP_SIZE}"
+fi
+
+export CLASSPATH=$HAZELCAST_HOME/hazelcast-enterprise-$HZ_VERSION/lib/hazelcast-enterprise-all-3.5.3.jar
```

Comment:
I should use $HZ_VERSION. Thanks now it is correct. Release script should only update $HZ_VERSION.